client/header/activity-panel/panels/inbox/action.js

```diff
@@ -69,7 +87,7 @@ class InboxNoteAction extends Component {
 				href={ action ? action.url : undefined }
 				onClick={ this.handleActionClick }
 			>
-				{ dismiss ? label : action.label }
+				{ this.getButtonLabel() }
```

Comment:
Should the actioned text be up one level in the `Card`?  https://github.com/woocommerce/woocommerce-admin/blob/ce68a4a48bac98d9da51ddd9b6c24cfb509c6f25/client/header/activity-panel/panels/inbox/card.js#L248

I think it might be best to display the text instead of the buttons, especially if there were multiple options.  There may also be some accessibility concerns around displaying buttons after actioning.

Reply:
I agree